ACCESSIBILITY.md

```diff
@@ -0,0 +1,97 @@
+
+In our Community Handbook, we describe these practices more specifically and in more detail. But as a baseline, our community is dedicated to enabling access across the various platforms we use through:
+- Providing diverse opportunities and platforms for people to contribute to _The Turing Way_ despite barriers or difficulties associated with one platform or another
+- Setting and embodying standards for writing and contribution that understand the way in which language may exclude as well as include people in data science, research, and other collaborative projects
+
+### 2.3 Community Event Accessibility
+
+As a community, we frequently organise events for a variety of purposes: for coworking, trainings, workshops, panels, conference talks, and more.
+
+In our Community Handbook, we describe these practices more specifically and in more detail. As a baseline however, we are committed to enabling accessibility through:
+- Supplying and recording transcripts for audio and closed captioning of video content
+- Enabling wide access to our program context across a variety of platforms
+- Uploading slides and/or presentation materials on Zenodo or other platforms to enable referencing
+
+### 3.1 How we work with Access-related Feedback
+
+Gathering access-related feedback is integral for ensuring that we can remain accessible as a community. We would like to offer a a few ways of sharing feedback with us, and are currently working to develop the following initiatives:
+
+1. **Feedback form**: Please submit access related through [this form](https://forms.gle/Ngr2eUtQmf7aEeao6). All of the information in this form is processed anonymously, with the option of submitting contact information by choice. 
+2. **Drop-in sessions**: Throughout the calendar year, we would like to facilitate drop-in session to receive feedback about access related different parts of project. These may take place at our Collaboration Cafe or may be hosted separately. 
+3. **Email the Research Community Manager or project team directly directly**: You can always email the Research Community Manager directly at asteele@turing.ac.uk with direct feedback. Alternatively, you can also 
+4. **Attend the Accessibility Working Group meeting**: The Accessibility Working Group meetings may be a place where you can deliver feedback in a small group environment. Please join the #Accessibility channel on slack for more information.
+
+We recognise that many feedback forms and other forms for gathering feedback can often operate as "blackboxes" that receieve information without taking actionable steps to address them.
+
+Knowing this, we are working to develop processes that enable us to action and report transparently on feedback given through these channels. This may include:
+- Reporting within the Accessibility Working Group in order to discuss possible actions that can be taken to address feedback
```

Comment:
are there links to these groups we can add? (I'm nervous of adding links given the accessibility points re embedded links so please ignore this and other comments if that advice applies)

Reply:
Great point, @harrietrs, we do have a broader page with some information: https://book.the-turing-way.org/afterword/subprojects.

Reply:
I have changed the link to markdown format so the description of the URL is what is read aloud.

Reply:
Thank you @PatriLoto and @LizHareDogs 🙏